packages/data-stores/src/plans/plans-data.ts

```diff
@@ -1,8 +1,55 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
 /**
  * Internal dependencies
  */
 import { PLAN_FREE, PLAN_PERSONAL, PLAN_PREMIUM, PLAN_BUSINESS, PLAN_ECOMMERCE } from './constants';
 
+export const PLANS_LIST = {
+	[ PLAN_FREE ]: {
+		title: translate( 'Free' ),
+		productId: 1,
+		storeSlug: PLAN_FREE,
+		pathSlug: 'beginner',
+		isPublicSite: false,
+	},
+
+	[ PLAN_PERSONAL ]: {
+		title: translate( 'Personal' ),
+		productId: 1009,
+		storeSlug: PLAN_PERSONAL,
+		pathSlug: 'personal',
+		isPublicSite: false,
+	},
+
+	[ PLAN_PREMIUM ]: {
+		title: translate( 'Premium' ),
+		productId: 1003,
+		storeSlug: PLAN_PREMIUM,
+		pathSlug: 'premium',
+		isPublicSite: false,
+	},
+
+	[ PLAN_BUSINESS ]: {
+		title: translate( 'Business' ),
+		productId: 1008,
+		storeSlug: PLAN_BUSINESS,
+		pathSlug: 'business',
+		isPublicSite: false,
+	},
+
+	[ PLAN_ECOMMERCE ]: {
+		title: translate( 'eCommerce' ),
+		productId: 1011,
+		storeSlug: PLAN_ECOMMERCE,
+		pathSlug: 'ecommerce',
+		isPublicSite: true,
```

Comment:
I don't expect for this flag to be added to a plans API. _public/private by default_ is some global system decision and not plan related. The ecommerce case is just an exception because some race condition in checkout 🙄